digos-ambassador/Content/Transformations/Species/snake/Arm.yml

```diff
@@ -0,0 +1,9 @@
+part: Arm
+species: snake
+description: "Transforms the target's {@side} arm into that of an anthro Serpent.”
```

Comment:
Don't capitalize Serpent.

Reply:
Weird, I thought I fixed that... wait I minute, I did. I must have uploaded an earlier version.